docs/source/python_api/mlflow.keras_core.rst

```diff
@@ -0,0 +1,7 @@
+mlflow.keras_core
+==================
+
+.. automodule:: mlflow.keras_core
+    :members:
+    :undoc-members:
+    :show-inheritance:
```

Comment:
https://output.circle-artifacts.com/output/job/a4f929b9-a0aa-484b-89cb-f98141afde19/artifacts/0/docs/build/html/python_api/mlflow.keras_core.html

<img width="774" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/ff0d745f-6517-4bfe-9060-a6c9c68080be">

Reply:
I think this is because my docstring is using Google style..?

Reply:
the doc is generated correctly: https://output.circle-artifacts.com/output/job/09b71378-edeb-4151-a65f-273aca51a8bc/artifacts/0/docs/build/html/python_api/mlflow.keras_core.html

But circleCI is still failing, odd

Reply:
sphinx tries to document `keras_core.src.callbacks.callback.Callback` because it's `MLflowCallback`'s parent.

```
/home/circleci/project/mlflow/keras_core/callback.py:docstring of mlflow.keras_core.callback.MLflowCallback:1: WARNING: py:class reference target not found: keras_core.src.callbacks.callback.Callback
```